packages/amplify-category-api/src/graphql-transformer/transform-graphql-schema.ts

```diff
@@ -4,6 +4,9 @@ import { $TSContext, ApiCategoryFacade } from '@aws-amplify/amplify-cli-core';
 import { AmplifyGraphQLTransformerErrorConverter } from '../errors/amplify-error-converter';
 import { transformGraphQLSchemaV1 } from './transform-graphql-schema-v1';
 import { transformGraphQLSchemaV2 } from './transform-graphql-schema-v2';
+import { forceRefreshSchema } from '../force-updates/force-refresh-schema';
+
+const FORCE_API_REFRESH_FLAG = 'force-api-refresh';
```

Comment:
Do we need new flag?
There's already `amplify push api --force` - should that just refresh schema?
See https://docs.amplify.aws/cli/commands/push/

Reply:
Updated to push `--force` to the `gql-compile` step, allowing us to keep from increasing the surface area on the options.